Gems/Atom/RPI/Code/Source/RPI.Edit/Material/MaterialSourceData.cpp

```diff
@@ -365,6 +365,13 @@ namespace AZ
 
             return Failure();
         }
+        
+        /*static*/ bool MaterialSourceData::LooksLikeImageFileReference(const MaterialPropertyValue& value)
```

Comment:
What's the `/*static*/` comment for?

Reply:
Copy/paste from the header. My personal style is to keep the static keyword commented out for clarity.

Reply:
Actually, if you search the codebase for "/*static*/" you'll see I'm not the only one who likes this.